specification/compute/resource-manager/readme.md

```diff
@@ -242,6 +243,20 @@ go:
   clear-output-folder: true
 ```
 
+### Go multi-api
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: package-compute-2017-03
+  - tag: package-container-service-2017-01

```

Comment:
Current recommendation is to ship "containerservice" outside of compute. These files are even deprecated, the new can be found here:
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/containerservices/resource-manager

Reply:
Ah, for the moment I am publishing [the union of API Versions that are present in both locations.](https://github.com/Azure/azure-sdk-for-go/tree/a49674fdba11173ce3bc934d020634e3366464de/services/containerservice/mgmt) There seems to be no overlap, so at the moment we don't have to choose how to resolve any conflicts.